Lib/email/utils.py

```diff
@@ -345,6 +346,8 @@ def localtime(dt=None, isdst=-1):
     to divine whether summer time is in effect for the specified time.
 
     """
+    if isdst != -1:
+        warnings._deprecated("The isdst parameter to localtime is deprecated.", remove=(3,13))
```

Comment:
```suggestion
        warnings._deprecated("The isdst parameter to localtime", remove=(3,13))
```

This function pads out the rest of the message for us:

```
_DEPRECATED_MSG = "{name!r} is deprecated and slated for removal in Python {remove}"
```
->

"The isdst parameter to localtime is deprecated and slated for removal in Python 3.13"